cli/src/main.rs

```diff
@@ -168,14 +168,17 @@ pub fn parse_args<'a>(
     let CliCommandInfo { command, signers } =
         parse_command(&matches, &default_signer, &mut wallet_manager)?;
 
-    let output_format = matches
+    let mut output_format = matches
         .value_of("output_format")
         .map(|value| match value {
             "json" => OutputFormat::Json,
             "json-compact" => OutputFormat::JsonCompact,
             _ => unreachable!(),
         })
         .unwrap_or(OutputFormat::Display);
+    if output_format == OutputFormat::Display && matches.is_present("verbose") {
+        output_format = OutputFormat::DisplayVerbose;
+    }
```

Comment:
It might be a little cleaner to change the above `unwrap_or()` to an `unwrap_or_else()` and move this logic inside